kubernetes/services/mac_manager.yaml

```diff
@@ -13,8 +13,7 @@ data:
     ignite.trust-store-path=truststore.jks
     ignite.trust-store-password=123456
     macmanager.oui = AA-BB-CC
-    macmanager.pool.size = 100
-    macmanager.retrylimit = 10
+    macmanager.retrylimit = 1000
```

Comment:
1000 retry? do we hit the limit in our perf test?

Reply:
No, but 1000 can ensure every mac address in range can be allocated.